### PR TITLE
[1612] change ShowHealthBarsComponent's DamageContainer field to a list

### PR DESCRIPTION
**Pull request by PrPleGoo**
Solves #1612

### CODE REVIEW

**Comment by Elijahrane** [comment]
Seems quite good. I'll have to test locally. This feature as a whole is getting close to being upstreamable.

File: Resources/Prototypes/Entities/Clothing/Eyes/hud.yml [hunk 1]
```diff
@@ -30,7 +31,8 @@
   - type: ClothingGrantComponent
     component:
     - type: ShowHealthBars
-      damageContainer: Biological
+      damageContainers: 
+      - Biological
```
Comment (Elijahrane) [inline Resources/Prototypes/Entities/Clothing/Eyes/hud.yml:35]: add the arachne one to any that currently have biological

Comment (PrPleGoo) [reply Resources/Prototypes/Entities/Clothing/Eyes/hud.yml:35]: Done

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Clothing/Eyes/hud.yml:35]: edit: disregard, just minor gh ui glitch

**Comment by Elijahrane** [comment]
I can confirm it works; please add half spirit to the medibot and medical robot ones, though.

**Review by Elijahrane — APPROVED** [review]

**Pull request merged**